src/runtime-rs/crates/hypervisor/src/dragonball/mod.rs

```diff
@@ -127,4 +126,8 @@ impl Hypervisor for Dragonball {
         let inner = self.inner.read().await;
         inner.get_jailer_root().await
     }
+
+    async fn save(&self) -> Option<HypervisorState> {
+        todo!("See https://github.com/kata-containers/kata-containers/issues/4674");
```

Comment:
Suggest change to comment `TOOD` the `todo!()` will cause panic.

Reply:
fixed